lms/djangoapps/teams/search_indexes.py

```diff
@@ -43,7 +44,15 @@ def data(self):
 
         Returns serialized object with additional search fields.
         """
-        serialized_course_team = CourseTeamSerializer(self.course_team).data
+        # Django Rest Framework v3.1 requires that we pass the request to the serializer
+        # so it can construct hyperlinks.  To avoid changing the interface of this object,
+        # we retrieve the request from the request cache.
+        context = {
+            "request": get_request_or_stub()
```

Comment:
I originally tried that approach, which required updating > 10 test cases.  Then I rebased onto master and saw a ton more failures because a PR had been merged that:

1) Fired a signal on save of a common model, which
2) Invoked a Python API that uses DRF, which
3) Raised an exception because the test case saved the model outside the context of a request.

I agree that this is ugly, but I think it's safer and more maintainable, since it avoids requiring every test to know about this peculiarity of DRF v3.1